x-pack/packages/kbn-ecs-data-quality-dashboard/impl/data_quality/data_quality_panel/pattern/index.tsx

```diff
@@ -227,6 +228,13 @@ const PatternComponent: React.FC<Props> = ({
           <EuiSpacer />
         </EuiFlexItem>
 
+        {!loading && pattern.includes(':') && (
```

Comment:
Do you know if the colon is reserved? Can users add it to any index patterns they may name? I could see it being used commonly.

Reply:
Tested it locally and looked it up. 👍🏾 
> Indices prior to 7.0 could contain a colon (:), but that’s been deprecated and won’t be supported in 7.0+